library/std/src/io/mod.rs

```diff
@@ -2856,6 +2857,12 @@ impl<T> Take<T> {
         self.limit
     }
 
+    /// Returns the number of bytes read so far.
+    #[stable(feature = "seek_io_take", since = "CURRENT_RUSTC_VERSION")]
+    pub fn position(&self) -> u64 {
```

Comment:
API needs to start out as `unstable` where possible, the only exception is trait implementations because that part doesn't currently work.

Reply:
I can't just change it to `unstable` because then the compiler complains that `seek_io_take` is both stable and unstable. I see a few options:
1. mark unstable the trait implementation
2. I have another feature for position
3. make position private (for now?)

Reply:
It can just be a separate feature gate

Reply:
I've changed it to be unstable and put behind a different feature gate `seek_io_take_position`.